src/java/org/apache/cassandra/schema/SystemDistributedKeyspace.java

```diff
@@ -68,6 +62,8 @@ private SystemDistributedKeyspace()
     {
     }
 
+    public static final String NAME = "system_distributed";
```

Comment:
is this ever used? I feel it is a duplicate of `SchemaConstants.DISTRIBUTED_KEYSPACE_NAME`